client/my-sites/checkout/src/components/wp-checkout-order-summary.tsx

```diff
@@ -52,6 +52,7 @@ import {
 import { getSignupCompleteFlowName } from 'calypso/signup/storageUtils';
 import { useSelector } from 'calypso/state';
 import { getCurrentPlan } from 'calypso/state/sites/plans/selectors';
+import { useGetWpcomPlanTotalIfPaidMonthly } from '../../utils';
```

Comment:
I am not a fun of having this single `utils` file where we dump everything in it. I would rather we place hooks under a `hooks/` folder. For this case, I think ok to have it under utils (`utils/use-wpcom-plan-total-if-paid-monthly.ts` again, the name strikes me as odd!).

Reply:
Done!